cmd/knoxite/main.go

```diff
@@ -57,31 +55,32 @@ var (
 		Short: "Knoxite is a data storage & backup tool",
 		Long: "Knoxite is a secure and flexible data storage and backup tool\n" +
 			"Complete documentation is available at https://github.com/knoxite/knoxite",
-		SilenceErrors: true,
-		SilenceUsage:  true,
+		SilenceErrors:     true,
+		SilenceUsage:      true,
 		DisableAutoGenTag: true,
 	}
 
-	logger knoxite.Logger
+	log knoxite.Logger
 )
 
 func main() {
 	shutdown.OnSignal(0, os.Interrupt, syscall.SIGTERM)
-	// quiet shutdown logger
-	shutdown.Logger = shutdown.LogPrinter(log.New(ioutil.Discard, "", log.LstdFlags))
+	// use quiet knoxite.NopLogger as shutdown logger
+	shutdown.Logger = shutdown.LogPrinter(knoxite.NopLogger{})
 	// shutdown.SetTimeout(0)
 
 	RootCmd.PersistentFlags().StringVarP(&globalOpts.Repo, "repo", "r", "", "Repository directory to backup to/restore from (default: current working dir)")
 	RootCmd.PersistentFlags().StringVarP(&globalOpts.Alias, "alias", "R", "", "Repository alias to backup to/restore from")
 	RootCmd.PersistentFlags().StringVar(&globalOpts.Password, "password", "", "Password to use for data encryption")
 	RootCmd.PersistentFlags().StringVarP(&globalOpts.ConfigURL, "configURL", "C", config.DefaultPath(), "Path to the configuration file")
-	RootCmd.PersistentFlags().StringVarP(&globalOpts.Verbosity, "verbose", "v", "Warning", "Verbose output: possible levels are Debug, Info and Warning")
+	RootCmd.PersistentFlags().StringVar(&globalOpts.LogLevel, "loglevel", "Print", "Verbose output. Possible levels are Debug, Info, Warning and Fatal")
+	RootCmd.PersistentFlags().BoolVarP(&globalOpts.Verbose, "verbose", "v", false, "Verbose output on log level Info. Use --loglevel to choose between Debug, Info, Warning and Fatal")
```

Comment:
Would probably be nice to have an increasing verbosity level from here using cobras `CountVarP()`. So using the `-v` flag results in loglevel `Info` and using `-vv` turns it into loglevel `Debug`.